package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "build": "npm run clean && better-npm-run build",
-    "build-all": "npm run clean && NODE_APP_INSTANCE=amo better-npm-run build && NODE_APP_INSTANCE=disco better-npm-run build",
+    "build-all": "npm run clean && NODE_APP_INSTANCE=amo better-npm-run build",
```

Comment:
is it still necessary to do `NODE_APP_INSTANCE=amo` when there's only one app? (if not this is the same as `build` now)

Reply:
it's still being used by `node-config` and other parts of our app.. I'll file an issue to try to get rid of it.

Reply:
https://github.com/mozilla/addons-frontend/issues/9807

Reply:
`build` works without it being defined?

Reply:
no, the readme says to invoke `build` with the env var